I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Home/Intro.rpy", line 10, in script
    $ AddStats("arouse", 100)
  File "game/Home/Intro.rpy", line 10, in <module>
    $ AddStats("arouse", 100)
  File "game/script.rpy", line 175, in AddStats
    renpy.show_screen("stat_animation", stat_name=stat_name, change_value=value)
  File "game/interface.rpy", line 32, in execute
    screen stat_animation(stat_name, change_value):
  File "game/interface.rpy", line 32, in execute
    screen stat_animation(stat_name, change_value):
  File "game/interface.rpy", line 36, in execute
    python:
  File "game/interface.rpy", line 45, in <module>
    renpy.log("ANIMATION: stat_name=%s, change_value=%s, image=%s" % (stat_name, change_value, ball_image))
TypeError: 'module' object is not callable

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Home/Intro.rpy", line 10, in script
    $ AddStats("arouse", 100)
  File "/Applications/renpy-8.3.7-sdk/renpy/ast.py", line 834, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-8.3.7-sdk/renpy/python.py", line 1187, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/Home/Intro.rpy", line 10, in <module>
    $ AddStats("arouse", 100)
  File "game/script.rpy", line 175, in AddStats
    renpy.show_screen("stat_animation", stat_name=stat_name, change_value=value)
  File "/Applications/renpy-8.3.7-sdk/renpy/display/screen.py", line 1362, in show_screen
    sls.add(_layer, d, _tag, zorder=_zorder, transient=_transient, keep_st=True, name=name)
  File "/Applications/renpy-8.3.7-sdk/renpy/display/scenelists.py", line 444, in add
    if not self.hide_or_replace(layer, remove_index, "replaced"):
  File "/Applications/renpy-8.3.7-sdk/renpy/display/scenelists.py", line 503, in hide_or_replace
    if not oldsle.displayable._handles_event(prefix):
  File "/Applications/renpy-8.3.7-sdk/renpy/display/screen.py", line 533, in _handles_event
    self.update()
  File "/Applications/renpy-8.3.7-sdk/renpy/display/screen.py", line 700, in update
    self.screen.function(**self.scope)
  File "game/interface.rpy", line 32, in execute
    screen stat_animation(stat_name, change_value):
  File "game/interface.rpy", line 32, in execute
    screen stat_animation(stat_name, change_value):
  File "game/interface.rpy", line 36, in execute
    python:
  File "game/interface.rpy", line 45, in <module>
    renpy.log("ANIMATION: stat_name=%s, change_value=%s, image=%s" % (stat_name, change_value, ball_image))
TypeError: 'module' object is not callable

macOS-26.0.1-arm64-arm-64bit arm64
Ren'Py 8.3.7.25031702
clicer 1.0
Sat Oct 25 15:17:19 2025
